source/CommandLine/CommandOutputProviderBase.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+using Octopus.CommandLine.OptionParsing;
+using Serilog;
+using Serilog.Events;
+
+namespace Octopus.CommandLine
+{
+    public abstract class CommandOutputProviderBase : ICommandOutputProvider
```

Comment:
A bunch of OctopusCLI specific things were left behind here as they weren't generic